### DISCUSSION TITLE
App crash on touch

### DISCUSSION THREAD

**Discussion by meznaric** [software-mansion/react-native-reanimated#3221] [Q&A]
Hey everyone,

I just upgraded react-native-reanimated to 2.8.0 from 1.7.0. We are using [callstack/repack](https://github.com/callstack/repack) for bundling which uses webpack under the hood (I don't think it matters) and reanimated babel plugin is used last.

I reduced our app to just a simple `<View />` with `<Text>` so none of the reanimated APIs are actually used.

React Native 0.68.1
Hermes: 0.11

App builds fine, javascript builds fine, there are no errors in any console, but when I **touch** the app anywhere it crashes.
`EventHandlerRegistry.cpp` line 52: `auto eventObject = jsi::Value::createFromJsonUtf8(`

Anyone has any ideas what could it be or what to debug next? 

`Thread 1: EXC_BAD_ACCESS (code=1, address=0x41b5d5fc)`
<img width="948" alt="Screenshot 2022-05-06 at 12 15 35" src="https://user-images.githubusercontent.com/1058131/167121569-03480fb9-84ae-4316-bad7-aca9cd77cbe6.png">

**Call stack:**
<img width="571" alt="Screenshot 2022-05-06 at 12 16 29" src="https://user-images.githubusercontent.com/1058131/167121570-b784f1c4-ec35-413d-99f6-cee3177b3c96.png">

**Local variables:**
<img width="321" alt="Screenshot 2022-05-06 at 12 18 38" src="https://user-images.githubusercontent.com/1058131/167121797-9c918d80-c7bd-450a-ae99-9f347847add8.png">

**Double checking in console:**
<img width="268" alt="Screenshot 2022-05-06 at 12 20 36" src="https://user-images.githubusercontent.com/1058131/167122059-d110d6db-7367-47eb-8347-d88105ec304d.png">

**Comment by quangnd1910** [comment]
Same

**Comment by paviad** [comment]
I also encounter this exact same problem - library version 2.8.0. Upgrading the library to 2.17.0 did not work, despite the code at the relevant line being wrapped in a `try` block - the error `EXC_BAD_ACCESS` is not a mere exception.
